lib/alg.ml

```diff
@@ -35,7 +35,8 @@ module FCFS_Ternary : Alg_t = struct
     {
       s = State.create 1;
       q = Pieotree.create topology;
-      z = scheduling_transaction;
+      z_in = scheduling_transaction;
+      z_out = fun x -> fun _ -> x;
```

Comment:
Feels to me like `fun s _ -> s` would be clearer?

Reply:
Done in [c049dd3](https://github.com/cucapra/schedulers-in-ocaml/pull/2/commits/c049dd3f22a39f5390cd2f0f6034fdbbfe74bea1)